lib/AST/ASTPrinter.cpp

```diff
@@ -1292,9 +1293,8 @@ void PrintAST::printPattern(const Pattern *pattern) {
 
   case PatternKind::Binding:
     if (!Options.SkipIntroducerKeywords)
-      Printer << (cast<BindingPattern>(pattern)->isLet() ? tok::kw_let
```

Comment:
Should we move checking `SkipIntroducerKeywords` into `printIntroducerKeyword`? That way you cannot forget, and also we could give the method the same signature as `printKeyword`, which also takes options.